Makefile

```diff
@@ -548,6 +548,7 @@ RUNTIME_CPP_COMPONENTS = \
   float16_t \
   gcd_thread_pool \
   gpu_device_selection \
+  hexagon_cpu_features \
```

Comment:
Turns out it's not anymore! It was needed when the runtime wasn't declared weak, another runtime module depended on this being available.